app/assets/javascripts/discourse/app/components/post-text-selection.gjs

```diff
@@ -64,14 +66,46 @@ export default class PostTextSelection extends Component {
   });
 
   documentListeners = modifier(() => {
-    document.addEventListener("mousedown", this.mousedown, { passive: true });
-    document.addEventListener("mouseup", this.mouseup, { passive: true });
+    // start of text selection
+    document.addEventListener("pointerdown", this.pointerdown, {
+      passive: true,
+    });
+
+    // any selection change
     document.addEventListener("selectionchange", this.onSelectionChanged);
 
+    // end of text selection on desktop, we also listen to
+    // it for touch devices due to desktop with touchscreen
+    document.addEventListener("pointerup", this.pointerup, { passive: true });
+
+    const { touch, isAndroid } = this.capabilities;
+    if (touch) {
+      if (isAndroid) {
+        // fires when user finishes adjusting text seleciton on android
+        // ios has no such event
```

Comment:
Could we just add the event listeners anyway? Or do they cause issues on other devices?

(Reducing code differences between devices makes things a lot easier to test/debug IMO)